packages/strapi-helper-plugin/lib/src/components/HeaderSearch/index.js

```diff
@@ -26,7 +26,7 @@ const HeaderSearch = ({ label, name, onChange, onClear, placeholder, value, clas
           </Clear>
         )}
       </div>
-      <SearchInfo label={label} />
+      {label && <SearchInfo label={label} />}
```

Comment:
Since the label in the defaultProps is `''` you should make the following statement:

```
import { isEmpty } from 'lodash';

{!isEmpty(label) && <SearchInfo label={label} />}
```

Reply:
As "" = false in JS, I think it is useless to import a function from a package to check if this string is empty or not.

Reply:
Sure when I tested it was the search was still working but I think it's because I forgot to build the helper.